OctaneVSPlugin/View/DetailsToolWindow.cs

```diff
@@ -65,7 +66,7 @@ public class DetailsToolWindow : ToolWindowPane
         /// </summary>
         public DetailsToolWindow() : base(null)
         {
-            this.Caption = "OctaneToolWindow";
+            Caption = "Loading entity...";
```

Comment:
I try to avoid displaying "entity" to the user, as only internal people get that terminology. The user would understand the type name or "backlog item". Either way I can't seem to find where this string is displayed, is it used?